hnn_core/gui/gui.py

```diff
@@ -417,6 +417,7 @@ def __init__(self, theme_color="#802989",
         # Add drive section
         self.drive_widgets = list()
         self.drive_boxes = list()
+        self.drive_accordion = Accordion()
```

Comment:
In the old code, was this information not stored persistently meaning you couldn't access the parent tree structure for deleting the drives?

Reply:
That's correct. It wasn't being stored. It was just being generated and displayed in the output. That happens here:
https://github.com/jonescompneurolab/hnn-core/blob/4abfa6f7da1eefff3af1d14a8be6d8103614f3db/hnn_core/gui/gui.py#L1467-L1480

Previously the only action we needed to do was add drives or delete them all. With add drive, the Accordion would just be rebuilt with the new drive and redisplayed in the output. With the delete function we need to access the Accordion to determine the selected accordion to close, which necessitated saving the Accordion to the class.